README.md

```diff
@@ -197,7 +197,73 @@ data-validation (--verbose or -v) (--log-level or -ll) validate row
+
+The command takes the same parameters as required for `Row Validation` *plus* few commands to implement the partitioning logic.
+
+`--partition-key`: Partitions are generated by adding filters on this key. If `--partition-key` is not supplied, it defaults to `--primary-keys`.
+
+(Note: As of now, only monotonically increasing key is supported for `--partition-key`.)
+
+`--partition-num`: Number of partitions or Config files to generate. Maximum is 1000. This parameter is truncated in case it exceeds the table count.
+
+`--config-dir`: Directory path to save the config files.
+
+```
+data-validation (--verbose or -v) (--log-level or -ll) generate-table-partitions
+
+  --source-conn or -sc SOURCE_CONN
+                        Source connection details
+                        See: *Data Source Configurations* section for each data source
+  --target-conn or -tc TARGET_CONN
+                        Target connection details
+                        See: *Connections* section for each data source
+  --tables-list or -tbls SOURCE_SCHEMA.SOURCE_TABLE=TARGET_SCHEMA.TARGET_TABLE
+                        Comma separated list of tables in the form schema.table=target_schema.target_table
+                        Target schema name and table name are optional.
+                        i.e 'bigquery-public-data.new_york_citibike.citibike_trips'
+  --primary-keys PRIMARY_KEYS, -pk PRIMARY_KEYS
+                        Comma separated list of primary key columns 'col_a,col_b'
+  --comparison-fields or -comp-fields FIELDS
+                        Comma separated list of columns to compare. Can either be a physical column or an alias
+                        See: *Calculated Fields* section for details
+  --hash COLUMNS        Comma separated list of columns to hash or * for all columns 
+  --concat COLUMNS      Comma separated list of columns to concatenate or * for all columns (use if a common hash function is not available between databases)
+  --config-dir CONFIG_DIR, -cdir CONFIG_DIR
+                        Directory Path to store YAML Config Files
+  --partition-num [1-1000], -pn [1-1000]
+                        Number of partitions/config files to generate
+                        In case this value exceeds the row count of the source/target table, its will be decreased to max(source_row_count, target_row_count)
+  [--partition-key PARTITION_KEY, -partkey PARTITION_KEY]
+                        Column on which the partitions would be generated. Column type must be integer. Defaults to Primary key
+  [--threshold THRESHOLD, -th THRESHOLD]
+                        Float max threshold for percent difference
+  [--filters SOURCE_FILTER:TARGET_FILTER]
+                        Colon spearated string values of source and target filters.
+                        If target filter is not provided, the source filter will run on source and target tables.
+                        See: *Filters* section
+  [--use-random-row, -rr]
```

Comment:
Supporting random row here wouldn't make sense since we want to break a full table into chunks. Random row would select rows and add a WHERE clause to select random rows; we want to cater this command to only full table row validations

Reply:
Done